python/python/bystro/tada_mixture/test/test_em.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from ..mixture_tada_em import MVTadaPoissonEM, MVTadaZPoissonEM
+
+
+def test_fit():
```

Comment:
I'm not sure I'm fully getting the idea behind this test.  Is there anything we want to check after fitting?  If not, would it make more sense to just fit the simplest matrix `X` that could possibly work?  It seems like we're setting X up with a lot of care that seems surprising relative to what we're testing [?]